### DISCUSSION TITLE
Update into new Spotify app

### DISCUSSION THREAD

**Discussion by IlyMooon** [SpotCompiled/SpotilifeC#5] [Ideas]
Hey, so ive downloaded th latest from the releases tab, and i saw it's the really old spotify from his look / features not avaible.
May we have the new primary update if it's available ?

**Comment by yodaluca23** [comment]
> Hey, so ive downloaded th latest from the releases tab, and i saw it's the really old spotify from his look / features not avaible.
> 
> May we have the new primary update if it's available ?

I believe, that 8.6.42 is the latest version that is supported by Spotilife, hopefully Julio will update it soon...

**Reply by bachig26** [reply]
> I believe, that 8.6.42 is the latest version that is supported by Spotilife, hopefully Julio will update it soon...

8.6.42 is more than 6 months old, I just saw 8.8.5 is the latest version. but I didn't know it's supported by Spotilife.
I also saw in the google search somewhere that the latest Spotilife version is 1.7, but not sure if it's legit version from Julio.

**Reply by yodaluca23** [reply]
> 8.6.42 is more than 6 months old, I just saw 8.8.5 is the latest version. but I didn't know it's supported by Spotilife.

It is not, otherwise I would've updated all SpotC repos… As soon as I learn of a newer version, that's compatible with all the tweaks then I will update it… 

> I also saw in the google search somewhere that the latest Spotilife version is 1.7, but not sure if it's legit version from Julio.

I don't know where you're getting that information but the latest version is 1.8 which is what I have on all of the SpotC repos.

**Reply by bachig26** [reply]
okay, thanks. yes I just saw 1.8...I just saw a site that came up when I tried to look for it because I didn't know the original site at the time.
